Vec2[] startPoint
Vec2[] endPoint
float64 width